drake/examples/Acrobot/test/paramEstSyntheticData.m

```diff
@@ -1,26 +1,99 @@
 function paramEstSyntheticData
 
-oldpath = addpath(fullfile(pwd,'..'));
+% Synthetic parameter estimation mode
+% 0: Base case parameter estimation - intial params = true params
+% 1: Try parameter estimation without any noise
+% 2: Try parameter estimation with measurement noise
+% 3: Try parameter estimation with noise and unknown delay (Not complete)
+mode = 2;
 
-r = AcrobotPlant;
-v = AcrobotVisualizer(r);
+% Method by which to obtain qdd
+% 0: Use acrobot dynamics to estimate true qdd
+% 1: Take the derivative of qd
+qddmode = 0;
 
-true_parameters = getParams(r);
+% Option to print from estimator: 0=noprint, 1=print
+% Option 0 will still print output of script
+parameterEstimationOptions.print_result = 0;
 
-%% test on swingup up data
-[utraj,xtraj] = swingUpTrajectory(r);
+% Standard deviation of the data input error
+% In this case, for theta1,theta2,theta1dot,theta2dot,theta1doubledot,theta2doubledot
+noisestd = sqrt([.0005, .0005, .0007, .0007, .0012, .0012,]);
+% Standard deviation of the parameter value percent error
+paramstd = 1/5;
 
-% Try parameter estimation without any noise
-Ts = .01; breaks=getBreaks(utraj); T0 = breaks(1); Tf = breaks(end);
-data = iddata(eval(xtraj,T0:Ts:Tf)',eval(utraj,T0:Ts:Tf)',Ts,'InputName',r.getInputFrame.getCoordinateNames(),'OutputName',r.getOutputFrame.getCoordinateNames());
+r = AcrobotPlant;
+rtrue = AcrobotPlant;
 
-estimated_parameters = parameterEstimation(r,data);
+if mode ~= 0
+    % Perturb original parameter estimates with random percentage error
+    % normally distributed with standard dev = paramstd, and greater than -1
```

Comment:
why the greater than -1 bit?  are you trying to make sure that the parameters do not go negative?  wouldn' t it be better to enforce that directly below?